external-crates/move/crates/move-trace-format/src/format.rs

```diff
@@ -171,7 +171,7 @@ pub enum TraceEvent {
         type_parameters: Vec<TypeTag>,
         pc: u16,
         gas_left: u64,
-        instruction: Box<String>,
+        instruction: Bytecode,
```

Comment:
We can talk about adding this in some way but...

Reply:
Like what ways? I'm open to discussion here.